pkg/client/query/supplierquerier.go

```diff
@@ -4,10 +4,9 @@ import (
 	"context"
 
 	"cosmossdk.io/depinject"
-	cosmosclient "github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/gogoproto/grpc"
```

Comment:
Just double-checking that we should in fact be using gogoproto and not vanilla grpc?

Reply:
`xxx.NewQueryClient` accepts the `grpc.ClientConn` interface from `"github.com/cosmos/gogoproto/grpc"`, while `grpc.ClientConn` from `"google.golang.org/grpc"` is a struct.

We could use `grpc.ClientConnInterface` from `"google.golang.org/grpc"` which seems to be the same.

WDYT?

Reply:
Sounds like you did the right thing already. 👍